### PR TITLE
Initial commit for python env in mlmd

### PR DESCRIPTION
**Pull request by varkha-d-sharma**
# Related Issues / Pull Requests

List all related issues and/or pull requests if there are any.

# Description

Include a brief summary of the proposed changes.

# What changes are proposed in this pull request?

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected; for instance, 
      examples in this repository need to be updated too).
- [ ] This change requires a documentation update.

# Checklist:

- [ ] My code follows the style guidelines of this project (PEP-8 with Google-style docstrings).
- [ ] My code modifies existing public API, or introduces new public API, and I updated or wrote docstrings that
      uses Google-style formatting and any other formatting that is supported by mkdocs and plugins this project
      uses.
- [ ] I have commented my code.
- [ ] My code requires documentation updates, and I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: cmflib/cmf.py [hunk 1]
```diff
@@ -1725,5 +1730,30 @@ def non_related_args(type:str,args:dict):
         if repo ==type:
             non_related_args=list(set(available_args)-set(dict_repository_args[repo]))
     return non_related_args
- 
-    
+
+
+def get_python_env()-> str:
```
Comment (annmary-roy) [inline cmflib/cmf.py:1735]: Make this a private function or better still ..add this in utils file

File: cmflib/cmf.py [hunk 2]
```diff
@@ -1725,5 +1730,30 @@ def non_related_args(type:str,args:dict):
         if repo ==type:
             non_related_args=list(set(available_args)-set(dict_repository_args[repo]))
     return non_related_args
- 
-    
+
+
+def get_python_env()-> str:
+    installed_packages = ""
+    python_version = sys.version
```
Comment (annmary-roy) [inline cmflib/cmf.py:1737]: What happens when both pip and conda is installed ?

File: cmflib/cmf.py (hunk 1 above)
Comment (varkha-d-sharma) [reply cmflib/cmf.py]: done

File: cmflib/cmf.py (hunk 2 above)
Comment (varkha-d-sharma) [reply cmflib/cmf.py]: conda gets preference.

**Review by annmary-roy — APPROVED** [review]

**Pull request merged**